nemo/collections/nlp/models/__init__.py

```diff
@@ -14,5 +14,5 @@
 
 from nemo.collections.nlp.models.lm_model import BERTLMModel
 from nemo.collections.nlp.models.qa_model import QAModel
-from nemo.collections.nlp.models.text_classification_model import TextClassificationModel
+from nemo.collections.nlp.models.text_classification.text_classification_model import TextClassificationModel
```

Comment:
yes, this should do the trick - in example you should be able now to do:

```from nemo.collections.nlp.models import TextClassificationModel```

Reply:
done